doc/gov/chapters/chapter01.md

```diff
@@ -4,48 +4,31 @@
 
 <a name="1.2"></a>
-## 1.2 Document Purpose
-
-The goal of this document is to assist CNTT to continue to scale, gain momentum, and plan for the future of CNTT-- outlined at the September 2019 3rd Technical Workshop held in Antwerp, Belgium. The purpose of this document is to define and detail basic CNTT community operations and important structures. It provides a reference of the organisation and related work streams, processes, and procedures around which the CNTT Community is managed; including operational scope, structure, and responsibilities of all groups and individuals associated with CNTT.  
-
-This document is intended for use by CNTT's new and existing contributors, participants, associated organisations, and, as an open-source community following the GSMA and LFN Anti-Trust Policies and Creative Commons Licensing, the public at-large.
-
-This document does not define any technical specifications, definitions, requirements, architectures, testing, certifications, or lab platforms. 
-
-<a name="1.3"></a>
-## 1.3 Operational Scope
-
-To support the technical mission and goals (link to docs here?), the scope of CNTT community operations are defined in this document as follows:
- 
-* Onboarding (Ch. 2)
-
-The CNTT Onboarding process ensures new members are informed of their obligations, confirms terms of agreement, and provides practical support to get them onboard as fully contributing members of the community.    
-
-* Organizational Structure and Works Streams (Ch. 3)
-
-This area includes the structure of governance and technical co-leadership and support, all participanting work streams and resources, the scope and the methods to manage the creation of deliverables by those work streams. 
-
-* Nomination and Selection (Ch. 4)
-
-Covers the processes for nominating and selecting candidates to be leads and co-leads for governance and technical workstreams, the timeframes of their service, responsibilities, and obiligations.
-
-* Consensus-Based Decision Making (Ch. 5)
-
-The approaches and processes CNTT contributors shall use to ensure fair representation for key decisions and release approvals. This includes mechanisms on how to manage dissension. 
-
-* Relationship to LFN Projects and GSMA (Ch. 6)
-
-Provides operational guidance for how CNTT operational structures shall interoperate with GSMA and LFN.
-
+## 1.2 How to participate
+
+Participating to Anuket does not need any formal membership, and nothing to sign except the CLA. Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you do automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 
```

Comment:
CLA needs expansion as well as a link and the other practices need a link.
Some minor typo corrections below
```suggestion
Participating in Anuket does not need any formal membership, and nothing to sign except the CLA. Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you do automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 
```

Reply:
Do we have approved CLA-s?